postgres/lib/PostgresService.js

```diff
@@ -249,18 +249,9 @@ GROUP BY k
       return super.from(from)
     }
 
-    // REVISIT: pg requires alias for {val}
-    SELECT_columns({ SELECT }) {
-      // REVISIT: Genres cqn has duplicate ID column
-      if (!SELECT.columns) return '*'
-      const unique = {}
-      return SELECT.columns
-        .map(x => `${this.column_expr(x)} as ${this.quote(this.column_name(x))}`)
-        .filter(x => {
-          if (unique[x]) return false
-          unique[x] = true
-          return true
-        })
+    column_alias4(x, q) {
+      if (!x.as && 'val' in x) return String(x.val)
```

Comment:
Is there a reason why `column_alias4` has to be overwritten here ? I did a local test run with it removed and the Postgres tests passed.